runtime/gc_base/GCExtensions.hpp

```diff
@@ -229,6 +229,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 		onStarted = 1,
 	};
 	TimingAddContinuationInList timingAddContinuationInList;
+	double testRAMSizePercentage;
```

Comment:
add a quick comment, like
/**< a factor to increase/decrease GC perceived physically memory size - for testing only */